hello-world/package.json

```diff
@@ -19,10 +20,13 @@
     ]
   },
   "dependencies": {
-    "temporalio": "0.5.0"
+    "temporalio": "0.4.2"
```

Comment:
Why is this change necessary? Might conflict with #29

Reply:
I don't remember. It did, and it's now 0.11.x 😄